planetiler-examples/README.md

```diff
@@ -63,31 +64,31 @@ at zoom level 12 and above:
 
 ```java
 @Override
-public void processFeature(SourceFeature sourceFeature, FeatureCollector features) {
-  if (sourceFeature.isPoint() && sourceFeature.hasTag("amenity", "toilets")) {
-    features.point("toilets") // create a point in layer named "toilets"
-      .setMinZoom(12)
-      .setAttr("customers_only", sourceFeature.hasTag("access", "customers"))
-      .setAttr("indoor", sourceFeature.getBoolean("indoor"))
-      .setAttr("name", sourceFeature.getTag("name"))
-      .setAttr("operator", sourceFeature.getTag("operator"));
+public void processFeature(SourceFeature sourceFeature,FeatureCollector features){
+  if(sourceFeature.isPoint()&&sourceFeature.hasTag("amenity","toilets")){
+  features.point("toilets") // create a point in layer named "toilets"
+  .setMinZoom(12)
+  .setAttr("customers_only",sourceFeature.hasTag("access","customers"))
+  .setAttr("indoor",sourceFeature.getBoolean("indoor"))
+  .setAttr("name",sourceFeature.getTag("name"))
+  .setAttr("operator",sourceFeature.getTag("operator"));
+  }
```

Comment:
looks like we lost the formatting here, if you're using intellij you can disable code autoformatting in markdown blocks using https://www.jetbrains.com/help/idea/markdown.html#disable-injection-in-code-blocks

Reply:
Formatting fixed.